package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "node-proxy-server",
+  "version": "1.0.0",
+  "description": "This is a project for learning Node/Express. In fact it will be proxy server for original NASA API.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
I'm not sure but probably it would be better to use some script to run the application
For example:
```
"scripts": {
    "dev": "node index.js"
 }
```

And then the run script can be used like this:
`npm run dev`

Reply:
Thanks. I've already added these changes to the following PR.